### PR TITLE
Update to support party arguments passed via the runtime enviornment.

### PR DESCRIPTION
**Pull request by mschaef-da**
* Accept parties via environment variable.
* Update the log message suite to reduce chattiness.
* Reduce default log levels, and provide a few mechanisms for controlling logging at runtime.
* Correct and improve documentation.

### CODE REVIEW

File: daml_dit_if/main/integration_context.py [hunk 1]
```diff
@@ -109,6 +110,7 @@ class IntegrationContext:
 
     def __init__(self,
                  network: 'Network',
+                 run_as_party: 'Optional[str]',
```
Comment (da-tanabe) [inline daml_dit_if/main/integration_context.py:113]: This really should be an `act_as`/`read_as` pair (and lists) to match the API for multi-party submissions. I anticipate being able to act as a member of a group as pretty important for integrations (and bots) going forward.

Comment (da-tanabe) [reply daml_dit_if/main/integration_context.py:113]: (and of course sadly you won't be able to feed this in the current version of dazl…but it's coming!)

Comment (lima-da) [reply daml_dit_if/main/integration_context.py:113]: ooo @da-tanabe this also implies artifact model better to separate `run_as` into `read_as/act_as` for all automation kinds (pybot, trigger, integrations)

Comment (mschaef-da) [reply daml_dit_if/main/integration_context.py:113]: Agreed... but... No. Not now. Later.

The major point of this PR is to make integrations  work like bots and triggers with respect to party arguments. That gives a better foundation for more broad-reaching changes like the one you mention here.

Breaking that to support a feature that doesn't yet exist in either DAZL or DABL seems very, very wrong.

https://github.com/DACH-NY/daml-cloud/blob/master/artifact-service/daml/DAMLHub/ArtifactService/V004/Companions/Records.daml#L128

Comment (mschaef-da) [reply daml_dit_if/main/integration_context.py:113]: @lima-da beat me to it. :-)

Comment (da-tanabe) [reply daml_dit_if/main/integration_context.py:113]: Well the reason why I bring it up now…are we changing the _external_ API that people are interacting with?

I'd rather change it once than twice (especially in the span of just a few weeks).

**Review by da-tanabe — APPROVED** [review]
Bleeeeaaaaaaaaaaah ok I yield. :-)

**Pull request merged**